src/AgendaMinutesGenerator.js

```diff
@@ -88,8 +95,10 @@ function _generateMain(
   Logger.log(JSON.stringify(data[0], " ", 2));
   fillTemplate(data, output_folder_id, template_id, title_formatter);
   ss.toast(
-    `${template_type} have been compiled!\nWrote:\n\t` +
-      data.map(title_formatter).join("\n\t")
+    `Wrote:\n\t` +
+      data.map(title_formatter).join("\n\t") +
+      ` to ${outputFolderLink}`,
+    `${template_type} have been compiled!`
```

Comment:
You can interpolate expressions too (at least, I think you can)

```javascript
    `Wrote:\n\t ${data.map(title_formatter).join("\n\t")} to ${outputFolderLink}`,
    `${template_type} have been compiled!`
```

Reply:
You definitely can - I think I was just leaving this line as is; will clean it up here: https://github.com/Mountain-View-Toastmasters/sign-up-sheet/pull/5